pwa-devdocs/scripts/create-env-variables-docs/handleVariable.js

```diff
@@ -17,7 +15,7 @@ const {
  * @returns {array} An array of mdast nodes containing the environment variable data
  */
 const handleVariable = variable => {
-  const {name, type, desc, example, default: defaultValue} = variable;
+  const {type, desc, example, default: defaultValue} = variable;
```

Comment:
I'd prefer this the other way (keep the destructure and just use `name` on L21) but that's super minor.